extensions/data-transfer/portability-data-transfer-koofr/src/main/java/org/datatransferproject/transfer/koofr/videos/KoofrVideosImporter.java

```diff
@@ -35,7 +35,9 @@
 import org.datatransferproject.types.common.models.videos.VideosContainerResource;
 import org.datatransferproject.types.transfer.auth.TokensAndUrlAuthData;
 
-/** Imports albums and videos to Koofr. */
+/**
+ * Imports albums and videos to Koofr.
```

Comment:
entirely-optional nit: just clarifying that it's not just generalized albums (even though it _could_ be, given the media vertical's complications)

```suggestion
 * Imports videos and their albums to Koofr.
```